.github/workflows/main.yml

```diff
@@ -216,14 +216,15 @@ jobs:
           go-version: 1.14
         id: go
 
-      - name: Run Simulation w/ jetstack/cert-manager, SMI policies (no Traffic Split), and egress disabled
+      - name: Run Simulation w/ jetstack/cert-manager, SMI policies (no Traffic Split), egress disabled, 1:N Service-ServiceAccount mapping
         env:
           CERT_MANAGER: "cert-manager" # enables jetstack/cert-manager integration
           BOOKSTORE_SVC: "bookstore-v1"
           BOOKTHIEF_EXPECTED_RESPONSE_CODE: "404"
           ENABLE_EGRESS: "false"
           EGRESS_EXPECTED_RESPONSE_CODE: "404" # egress is disabled
           DEPLOY_TRAFFIC_SPLIT: "false"
+          DEPLOY_WITH_SAME_SA: "true"
```

Comment:
I don't even know if we need the optionality.  I'd say the various versions of the bookstore could *always* live in the same SA.

Reply:
We also want to ensure our code works when the different versions are on different ServiceAccounts. So we have a few combinations tested in the CI.